nms/app/packages/magmalte/app/views/equipment/EquipmentGateway.js

```diff
@@ -349,15 +322,15 @@ function GatewayStatusTable(props: WithAlert & {refresh: boolean}) {
         numEnodeBs = gateway.connected_enodeb_serials.length;
       }
 
-      let checkInTime = new Date(0);
+      let checkInTime = '-';
```

Comment:
I'm assuming you're doing this so you can display checkin time as "-" if it doesn't exist (correct me if I'm wrong). I don't think you should change the type (above) just to allow that to happen this way. Instead, keep it as a `Date` and then when you go to display it check if the `checkinTime == Date(0)` or something and then display whatever you want there.

Reply:
checkinTime is an row attribute taken by the MaterialTable, it takes in an array of rows as a prop and displays them on the table. If I keep the attribute as Date, I end up with flow errors, so modified it this way.